test/basic_server.js

```diff
@@ -1,87 +1,103 @@
+    });
 
-  server.auth.strategy('jwt-nourl2', 'jwt', {
-    key: secret,
-    validateFunc: validate,
-    verifyOptions: { algorithms: [ 'HS256' ] }, // only allow HS256 algorithm
-    urlKey: ''
-  });
+    server.auth.strategy('jwt-nocookie2', 'jwt', {
+      key: secret,
+      validate,
+      verifyOptions: { algorithms: [ 'HS256' ] }, // only allow HS256 algorithm
+      cookieKey: ''
+    });
 
-  server.auth.strategy('jwt-nocookie2', 'jwt', {
-    key: secret,
-    validateFunc: validate,
-    verifyOptions: { algorithms: [ 'HS256' ] }, // only allow HS256 algorithm
-    cookieKey: ''
-  });
+    server.auth.strategy('jwt-headless', 'jwt', {
+      key: secret,
+      validate,
+      verifyOptions: { algorithms : ['HS256'] },
+      headless: { alg: 'HS256', typ: 'JWT' }
+    });
 
-  server.route([
-    { method: 'GET',  path: '/', handler: home, config: { auth: false } },
-    { method: 'GET', path: '/token', handler: sendToken, config: { auth: 'jwt' } },
-    { method: 'POST', path: '/privado', handler: privado, config: { auth: 'jwt' } },
-    { method: 'POST', path: '/privadonourl', handler: privado, config: { auth: 'jwt-nourl' } },
-    { method: 'POST', path: '/privadonocookie', handler: privado, config: { auth: 'jwt-nocookie' } },
-    { method: 'POST', path: '/privadonourl2', handler: privado, config: { auth: 'jwt-nourl2' } },
-    { method: 'POST', path: '/privadonocookie2', handler: privado, config: { auth: 'jwt-nocookie2' } },
-    { method: 'POST', path: '/required', handler: privado, config: { auth: { mode: 'required', strategy: 'jwt' } } },
-    { method: 'POST', path: '/optional', handler: privado, config: { auth: { mode: 'optional', strategy: 'jwt' } } },
-    { method: 'POST', path: '/try', handler: privado, config: { auth: { mode: 'try', strategy: 'jwt' } } }
-  ]);
+    server.route([
+      { method: 'GET', path: '/', handler: home, config: { auth: false } },
+      { method: 'GET', path: '/token', handler: sendToken, config: { auth: 'jwt' } },
+      { method: 'POST', path: '/privado', handler: privado, config: { auth: 'jwt' } },
+      { method: 'POST', path: '/privadonourl', handler: privado, config: { auth: 'jwt-nourl' } },
+      { method: 'POST', path: '/privadonocookie', handler: privado, config: { auth: 'jwt-nocookie' } },
+      { method: 'POST', path: '/privadonourl2', handler: privado, config: { auth: 'jwt-nourl2' } },
+      { method: 'POST', path: '/privadonocookie2', handler: privado, config: { auth: 'jwt-nocookie2' } },
+      { method: 'POST', path: '/required', handler: privado, config: { auth: { mode: 'required', strategy: 'jwt' } } },
+      { method: 'POST', path: '/optional', handler: privado, config: { auth: { mode: 'optional', strategy: 'jwt' } } },
+      { method: 'POST', path: '/try', handler: privado, config: { auth: { mode: 'try', strategy: 'jwt' } } },
+      { method: 'POST', path: '/headless', handler: privado, config: { auth:  'jwt-headless' } }
+    ]);
+  } catch(e) {
```

Comment:
I don't _think_ there is a need to `catch` and then `throw` the error, 
but it's fine. as it makes it  _clearer_. 👍